### PR TITLE
Address feedback and TODOs for "Build Node" guide

### PR DESCRIPTION
**Pull request by jkczyz**

### CODE REVIEW

File: docs/build_node.md [hunk 1]
```diff
@@ -222,6 +222,10 @@ final channel_manager = ((Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ
     .res.b;
 ```
 
+**Implementation notes:** No methods should be called on `ChannelManager` until
+after it has been synced and it's `chain::Watch` has been given the
```
Comment (valentinewallace) [inline docs/build_node.md:226]: nit: its

File: docs/build_node.md [hunk 2]
```diff
@@ -378,14 +378,22 @@ while (true) {
 
 **Dependencies:** `ChannelManager`, `ChainMonitor`
 
-### Once Per Minute: `ChannelManager`'s `timer_chan_freshness_every_min()`
-**What it's used for:** `ChannelManager` needs to be told every time a minute passes so that it can broadcast fresh channel updates if needed
+### Background Processing
+
+`ChannelManager`'s `timer_chan_freshness_every_min()`
```
Comment (valentinewallace) [inline docs/build_node.md:383]: Could we remove this line? I'd like to keep the format of the steps consistent, so 1) name of step, 2) what it's used for, 3) example, etc, and this line breaks that.

File: docs/build_node.md [hunk 3]
```diff
@@ -378,14 +378,22 @@ while (true) {
...
+
+**What it's used for:** running tasks periodically that aren't high-priority, e.g.,
```
Comment (valentinewallace) [inline docs/build_node.md:385]: Could we take out the `e.g.` -- `.e.g.` means "for example," and those two steps aren't examples, they're the entire step.

File: docs/build_node.md (hunk 2 above)
Comment (jkczyz) [reply docs/build_node.md]: Whoops, meant to delete that.

**Review by valentinewallace — APPROVED** [review]
Awesome thanks @jkczyz, great to get Igor's feedback in

**Pull request merged**